cleanlab/regression/rank.py

```diff
@@ -0,0 +1,24 @@
+import numpy as np 
+
+def get_label_quality_score(
+    true_labels: np.ndarray,
+    pred_labels: np.ndarray
+)-> np.ndarray:
+    """
+    Returns label quality score
```

Comment:
```suggestion
    Returns label quality score for each example in a regression dataset.
```

Reply:
Docstring modified in commit: [0503595](https://github.com/krmayankb/cleanlab/pull/1/commits/05035950d782fee4adde8da6c44afb47953fb5db)